instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -250,10 +264,12 @@ def _handle_exception(
 
     def __call__(self, env, start_response):
         # pylint: disable=E1101
+        # pylint: disable=too-many-locals
         if self._otel_excluded_urls.url_disabled(env.get("PATH_INFO", "/")):
             return super().__call__(env, start_response)
 
         start_time = time_ns()
+        start = default_timer()
```

Comment:
Should the timer start right before the `return super().__call__(env, _start_response)` below to measure the actual duration of only the request?

Reply:
Shouldn't you be removing this line from here since it's already added below.